README.md

```diff
@@ -1 +1,15 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+Built With
+
+    HTML
+    CSS
+
+
```

Comment:
- [ ] [OPTIONAL] IT would be better if you add a section where you tell how the users should install your project locally